server/views/partials/viewCases.njk

```diff
@@ -0,0 +1,130 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "./statusBadge.njk" import statusBadge %}
+
+{% set pageTitle = applicationName + " - view an approved licence hardstop" %}
+{% set pageId = 'view-hardstop-cases-page' %}
+
+{% macro viewCases(cases, isHardstop, statusConfig) %}
+    {% set cases = cases | beforeOrAfterHardStop(isHardstop) %}
+    {% set licences = [] %}
+    {% macro offenderName(offenderName, prisonerNumber, licenceId, isClickable, versionOfId, licenceStatus, index) %}
+        <div class="caseload-offender-name">
+            {% if isClickable %}
+                {% set link = "/licence/view/id/" + licenceId + "/show" %}
+                {% if versionOfId and licenceStatus === 'SUBMITTED' %}
+                    {% set link = link + "?lastApprovedVersion=" + versionOfId %}
+                {% endif %}
+                <a id="name-button-{{ index }}" href={{ link }} class="govuk-link govuk-heading-s">{{ offenderName }}</a>
+            {% else %}
+                <span id="name-button-{{ index }}" class="govuk-heading-s">{{ offenderName }}</span>
+            {% endif %}
+        </div>
+    {% endmacro %}
+    {% for case in cases %}
+        {% if case.probationPractitioner %}
+            {% set comDetailsHtml = '<a class="govuk-link" href="/licence/view/probation-practitioner/staffCode/' + case.probationPractitioner.staffCode + '">' + case.probationPractitioner.name + '</a>' %}
+        {% else %}
+            {% set comDetailsHtml = 'Unallocated' %}
+        {% endif %}
+        {% if case.licenceVersionOf and (case.licenceStatus === 'IN_PROGRESS' or case.licenceStatus === 'SUBMITTED') %}
+            {% set licenceStatusHtml = statusBadge({ status: case.licenceStatus, size: 'small' }, statusConfig) +
+                                    "<div class='govuk-!-margin-top-2'><svg class='moj-banner__icon' fill='currentColor' role='presentation'" +
```

Comment:
not sure if it existed before but would suggest extracting this svg into its own macro with a good name rather than embedding within a string within a set

Reply:
is there a reason why you aren't using the component? 
https://design-patterns.service.justice.gov.uk/components/banner/

(it's very difficult to understand what's going on here due to the complexity of the nesting - might help to break it into separate macros)